packages/base-map/src/styled.ts

```diff
@@ -103,11 +100,15 @@ export const LayerSelector = styled.aside`
     &:hover,
     &.fake-mobile-hover {
       label {
-        display: block;
```

Comment:
I *think* this line is needed to override other styles that are applied -- even though it's defined above.

Reply:
`display:block` is still in effect on the client (see screenshot).

<img width="573" alt="image" src="https://user-images.githubusercontent.com/56846598/236552099-90db7254-b54b-4558-8f32-dc08aad302c4.png">